tests/backend/test_flow_run.py

```diff
@@ -600,11 +601,32 @@ def test_watch_flow_run_timeout(monkeypatch):
     # Mock sleep so that we do not have a slow test
     monkeypatch.setattr("prefect.backend.flow_run.time.sleep", MagicMock())
 
-    with pytest.raises(RuntimeError, match="timed out after 12 hours of waiting"):
+    with pytest.raises(RuntimeError, match="timed out after 12.0 hours of waiting"):
```

Comment:
Maybe add a comment here indicating that you're testing the default behavior?

Reply:
Good point